__tests__/app.test.js

```diff
@@ -256,5 +256,41 @@ describe('endpoint GET /api/users', ()=>{
             expect(body.users).toEqual(testData.userData)
         })
     })
+})
 
+describe('endpoint GET /api/articles (topic query)', ()=>{
+    test('receives 200 response and responds with articles filtered by the topic value specified in the query', ()=>{
+        return request(app)
+        .get('/api/articles?topic=cats')
+        .expect(200)
+        .then(({body})=>{
+            expect(body.articles).toEqual( [{
+                article_id:5,
+                title: "UNCOVERED: catspiracy to bring down democracy",
+                topic: "cats",
+                author: "rogersop",
+                comment_count: "2",
+                created_at: expect.any(String),
+                votes: 0,
+                article_img_url:
+                  "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+              }])
+        })
```

Comment:
it would be good to include a test for a topic that returns multiple articles, such as `mitch`